admin/build_scientific_colors_cpt.sh

```diff
@@ -38,51 +39,79 @@ DIR=$1
 VERSION=6.0.4
 cat << EOF > /tmp/cpt.info
 acton|Perceptually uniform sequential colormap, by Fabio Crameri [C=RGB]
+actonS|Perceptually uniform sequential categorical colormap, by Fabio Crameri [C=RGB]
 bamako|Perceptually uniform, low-lightness gradient colormap by Fabio Crameri [C=RGB]
+bamakoS|Perceptually uniform, low-lightness gradient categorical colormap by Fabio Crameri [C=RGB]
 batlow|Perceptually uniform ‘rainbow’ colormap by Fabio Crameri [C=RGB]
+batlowS|Perceptually uniform ‘rainbow’ categorical colormap by Fabio Crameri [C=RGB]
```

Comment:
Is there any special reason you're using the non-ASCII single quotes `‘` vs `'`?

Reply:
I am using the C syntax \' in the text.  I think \' and \" gives the right quotes, no?

Reply:
`'` and `"` are both correct, but the quotes in the following line is non-ASCII. 
```
batlowS|Perceptually uniform ‘rainbow’ categorical colormap by Fabio Crameri [C=RGB]
```

Reply:
Sorry formatting in the comment.  Per C manual I can use

`\' and \" for quotes`

which is what I do. I am not using` ‘ `

Reply:
I'm talking about the `admin/build_scientific_colors_cpt.sh` bash script and the generated CPTs. 

If you go to the share/cpt directory, and run `file *.cpt`, you will see while most CPTs are ASCII plaintext files, some are not, due to the non-ASCII `‘`.
```
abyss.cpt:       ASCII text
acton.cpt:       ASCII text
bamako.cpt:      ASCII text
bathy.cpt:       ASCII text
batlow.cpt:      UTF-8 Unicode text
berlin.cpt:      ASCII text
bilbao.cpt:      ASCII text
broc.cpt:        ASCII text
```

Reply:
Yes, sorry, I was looking in gmt_cpt_masters.h....
I see what you mean and will fix, and also rename the build script to replace those underscores with hyphens, like our other build scripts.